app/models/game.rb

```diff
@@ -0,0 +1,10 @@
+class Game < ApplicationRecord
+  belongs_to :user
+  has_many :rounds
+  validates :game_mode, presence: true
+  validates :user_id, presence: true
+  validates :score, presence: true
```

Comment:
no validation for score because we get the score after we started the game

Reply:
it should have a default value at creation of the class instance. I'm gonna add it now